contracts/bridge/contracts/binance/SubstrateMessageCommitment.sol

```diff
@@ -0,0 +1,177 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.6.0 <0.7.0;
+pragma experimental ABIEncoderV2;
+
+import "@darwinia/contracts-verify/contracts/MerkleProof.sol";
+import "../interfaces/ILightClientBridge.sol";
+
+contract SubstrateMessageCommitment {
```

Comment:
```suggestion
contract BeefyMessageCommitment {
```

Reply:
I consider if it's better to change `BeefyMessageCommitment` to common `TargetChainMessageCommitmemt` rather than a `BeefyMessageCommitment`, A more appreciate `BeefyMessageCommitment` can be this
```
struct commitment {
    payload: Payload
    blockNumber: uint32
    validatorSetId: uint64
}

struct Payload {
    messageRoot: bytes32
    mmr: bytes32
    beefyNextAuthoritySet: BeefyNextAuthoritySet
}
```
And we only care about message-proof to `meesageRoot`at message layer, no more deal with mmrRoot or mmr-proof.